app/models/event_category.rb

```diff
@@ -1,6 +1,6 @@
 class EventCategory < ApplicationRecord
   belongs_to :event
-  has_many :tickets
+  has_many :tickets, dependent: :destroy
```

Comment:
not sure we want this here either -- in general we don't want tickets and orders to be deleted because they represent a "transaction" with a 3rd party (we issue only "sold" tickets)